### ISSUE TITLE
AnP command to move allocations

### ISSUE THREAD

**Issue by jasonwalker80** [genome/genome#1691]
The new command would move ALL allocations for an AnP from one disk group to another. The implementation should allow for the move of individual models/builds as well.

See #1687 for a discussion of `disk_usage_allocations`. Once implemented for all model types using a list of subclass names that "belong" to a model, this new command should be fairly straightforward when an allocation is unique to an AnP.

**Comment by chrisamiller** [comment]
Wondering if you've given any thought to this yet.  We've got like dozen models that are in the wrong disk_group due to confusion early on. Just don't want anyone to get billed for data storage that their project isn't actually using. I could go through and move the bam files using techniques we've talked about previously, but I'd rather be able to do a clean move of the whole model so that things don't get confusing.
